internal/cmd/gocdk/_static/init/.dockerignore

```diff
@@ -1,3 +1,5 @@
+/biomes/
```

Comment:
Why ignore `biomes`? Don't we expect that to be checked in?

Reply:
Yes, but `.dockerignore` is about what data is copied from the working copy to the Docker daemon, not what is checked into version control. Biome files are never needed to produce the built Docker image.

Reply:
Ah, sorry, I missed that this was `.dockerignore` not `.gitignore`. Updating title.